courses/models.py

```diff
@@ -14,6 +14,7 @@ class Program(models.Model):
     title = models.CharField(max_length=255)
     live = models.BooleanField(default=False)
     description = models.TextField(blank=True, null=True)
+    personalized_pricing = models.BooleanField(default=False)
```

Comment:
I did a similar change in #1019 and I think we should stick to that name. 
If you prefer to keep this here in this PR, please rename the field (in the django model, I do not care for the rest) and I can remove this part from my PR, otherwise we can do the opposite, just let me know.

Sorry for the confusion.